### PR TITLE
bug(VSelect): Removed line at 180 from VMenu.ts. bug #14489

### PR DESCRIPTION
**Pull request by adil-portable**
Removed line 180 from VMenu.ts which was unnecessarily setting v-list-item--highlighted. 

## Description

Removed line 180 from VMenu.ts which was unnecessarily setting v-list-item--highlighted. 

## Motivation and Context

bug #14489
## How Has This Been Tested?

It was tested manually(visually) and by running cypress tests.

It updated the existing ones.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here

<template>
  <v-app id="inspire">
    <v-container fluid>
      <v-select
        v-model="selectedFruits"
        :items="fruits"
        label="Favorite Fruits"
        multiple
      >
        <template #prepend-item>
          <v-list-item
            ripple
            @click="toggle"
          >
            <v-list-item-action>
              <v-icon :color="selectedFruits.length > 0 ? 'indigo darken-4' : ''">
                {{ icon }}
              </v-icon>
            </v-list-item-action>
            <v-list-item-content>
              <v-list-item-title>
                Select All
              </v-list-item-title>
            </v-list-item-content>
          </v-list-item>
          <v-divider class="mt-2"></v-divider>
        </template>
        <template #append-item>
          <v-divider class="mb-2"></v-divider>
          <v-list-item disabled>
            <v-list-item-avatar color="grey lighten-3">
              <v-icon>
                mdi-food-apple
              </v-icon>
            </v-list-item-avatar>
            <v-list-item-content v-if="likesAllFruit">
              <v-list-item-title>
                Holy smokes, someone call the fruit police!
              </v-list-item-title>
            </v-list-item-content>
            <v-list-item-content v-else-if="likesSomeFruit">
              <v-list-item-title>
                Fruit Count
              </v-list-item-title>
              <v-list-item-subtitle>
                {{ selectedFruits.length }}
              </v-list-item-subtitle>
            </v-list-item-content>
            <v-list-item-content v-else>
              <v-list-item-title>
                How could you not like fruit?
              </v-list-item-title>
              <v-list-item-subtitle>
                Go ahead, make a selection above!
              </v-list-item-subtitle>
            </v-list-item-content>
          </v-list-item>
        </template>
      </v-select>
    </v-container>
  </v-app>
</template>

<script>
  export default {
    data: () => ({
      fruits: [
        'Apples',
        'Apricots',
        'Avocado',
        'Bananas',
        'Blueberries',
        'Blackberries',
        'Boysenberries',
        'Bread fruit',
        'Cantaloupes (cantalope)',
        'Cherries',
        'Cranberries',
        'Cucumbers',
        'Currants',
        'Dates',
        'Eggplant',
        'Figs',
        'Grapes',
        'Grapefruit',
        'Guava',
        'Honeydew melons',
        'Huckleberries',
        'Kiwis',
        'Kumquat',
        'Lemons',
        'Limes',
        'Mangos',
        'Mulberries',
        'Muskmelon',
        'Nectarines',
        'Olives',
        'Oranges',
        'Papaya',
        'Peaches',
        'Pears',
        'Persimmon',
        'Pineapple',
        'Plums',
        'Pomegranate',
        'Raspberries',
        'Rose Apple',
        'Starfruit',
        'Strawberries',
        'Tangerines',
        'Tomatoes',
        'Watermelons',
        'Zucchini',
      ],
      selectedFruits: [],
    }),
    computed: {
      likesAllFruit () {
        return this.selectedFruits.length === this.fruits.length
      },
      likesSomeFruit () {
        return this.selectedFruits.length > 0 && !this.likesAllFruit
      },
      icon () {
        if (this.likesAllFruit) return 'mdi-close-box'
        if (this.likesSomeFruit) return 'mdi-minus-box'
        return 'mdi-checkbox-blank-outline'
      },
    },

    methods: {
      toggle () {
        this.$nextTick(() => {
          // eslint-disable-next-line no-debugger
          debugger
          if (this.likesAllFruit) {
            this.selectedFruits = []
          } else {
            this.selectedFruits = this.fruits.slice()
          }
        })
      },
    },
  }
</script>

```

</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VMenu/VMenu.ts [hunk 1]
```diff
@@ -177,7 +177,6 @@ export default baseMixins.extend({
     listIndex (next, prev) {
       if (next in this.tiles) {
         const tile = this.tiles[next]
-        tile.classList.add('v-list-item--highlighted')
```
Comment (KaelWD) [inline packages/vuetify/src/components/VMenu/VMenu.ts:180]: This is used for keyboard navigation to indicate the focused item.

File: packages/vuetify/src/directives/ripple/index.ts [hunk 2]
```diff
@@ -204,7 +204,7 @@ function rippleHide (e: Event) {
     element._ripple.showTimerCommit = null
 
     // re-queue ripple hiding
-    element._ripple.showTimer = setTimeout(() => {
+    element._ripple.showTimer = window.setTimeout(() => {
```
Comment (KaelWD) [inline packages/vuetify/src/directives/ripple/index.ts:207]: Please do not make unrelated changes to other files.

**Review by KaelWD — CHANGES_REQUESTED** [review]

File: packages/vuetify/src/directives/ripple/index.ts (hunk 2 above)
Comment (adil-portable) [reply packages/vuetify/src/directives/ripple/index.ts:207]: I updated this line because it was throwing an error (Type 'Timeout' is not assignable to type 'number'.ts(2322)
) due to which I was unable to push my changes.
